.werft/jobs/build/job-config.ts

```diff
@@ -179,3 +181,23 @@ function parseVersion(context: any) {
     }
     return version;
 }
+
+export function parseWithIntegrationTests(werft: Werft, sliceID: string, value?: string): WithIntegrationTests {
+    switch (value) {
+        case null:
+        case undefined:
+            return "skip";
```

Comment:
Why do we default to `skip` here, but for undefined/unknown we default to `all`?

Reply:
It's mainly about how we want to interpret how people use (or not use) the `with-integration-tests` job attribute, especially when used in GH comments/PR descriptions where you can use `/werft run with-integration-tests` or if you just set the attribute in your PR description (`/werft with-integration-tests` - in both cases without specifying a value -  which case the value is the empty string.

In the CLI it's less common but you could do `werft job run github -a with-integration-tests=""`.

So if it is null/undefined it means they didn't use the `with-integration-tests` attribute at all - if it's the empty string they did use it, but just didn't specify a value - I decided to interpret that as "they want to run all the tests"

Reply:
Would it be useful to add this explanation as a comment in the code, or do you think it is overkill?

Reply:
I added a log line of

```
werft.log(sliceID, "with-integration-tests was not set - will use 'skip'");
```

As I think that will

- Capture the essence of the description above. Each of the three cases (not set, set to empty, set to unknown value) now have a log line that describe things a bit
- This might be useful as it will help people know if they made a typo in the attribute, e.g. "with-integration-test" (missing `s`)